eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           /p:Configuration=$(BuildConfiguration)
           /p:CollectCoverage=$(CollectCoverage) /p:CodeCoverageDirectory=$(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}
           /p:EnableSourceLink=false 
-          /p:ProjectListOverrideFile='${env: parameters.ProjectListOverrideFilePropertyName }'
+          /p:ProjectListOverrideFile=$env: ${{parameters.ProjectListOverrideFilePropertyName }}
```

Comment:
Does this work with the space?

Reply:
No. Try many different ways. Not working well in our case.

We have two cases:
1. `$(ProjectListOverrideFilePropertyName)`
2. Empty value. 
I did not find a way to deserialize the cases into right values.

Reply:
yeah, in devops yaml you cannot do a second pass evaluation, i.e. where a parameter renders to variable reference, which then renders to a value.